src/tools/annotation/CircleRoiTool.js

```diff
@@ -507,6 +510,17 @@ function _calculateStats(image, element, handles, modality, pixelSpacing) {
       ((pixelSpacing && pixelSpacing.colPixelSpacing) || 1)) /
     2;
   const perimeter = 2 * Math.PI * radius;
+
+  // Pixel diagonal
+  const pixelDiagonal = pixelSpacing.colPixelSpacing
+    ? decimal(
+        pixelSpacing.colPixelSpacing * pixelSpacing.colPixelSpacing +
+          pixelSpacing.rowPixelSpacing * pixelSpacing.rowPixelSpacing
+      ).sqrt()
+    : decimal.sqrt(2);
```

Comment:
Instead of calculating sqrt(2) every time we could have it stored in a constant 🤔